in_toto/verifylib.py

```diff
@@ -1070,11 +1070,11 @@ def verify_sublayouts(layout, chain_link_dict):
 
   for step_name, key_link_dict in six.iteritems(chain_link_dict):
 
+    # Extract keyid and link from the passed chain_link_dict
```

Comment:
This comment states the obvious (i.e. something that can be easily gathered from the code); and contradicts the code (which iterates over keyid, link pairs in a key_link_dict). Both bad commenting practices. Please re-phrase.